requirements.txt

```diff
@@ -1,7 +1,8 @@
 numpy
 scipy
 matplotlib
-pyqt5
+pyqt5>=5.10,<5.14; platform_system == "Darwin"
+pyqt5>=5.10; platform_system == "Darwin"
```

Comment:
@larsoner should this be != ?

Reply:
Whoops, yes! Can you open a quick PR?

Reply:
Actually I can push a quick commit to fix it

Reply:
Sorry I should have just opened a pr to start with.